book/src/overview/how-does-harbor-fit-in.md

```diff
@@ -0,0 +1,18 @@
+## How does Harbor fit in?
+
+Unlike automobile manufacturing, software development and procurement is happening in virtually 
```

Comment:
software development, maintenance and procurement... Rule of three: https://jerichowriters.com/the-rule-of-three-in-writing/

Not that you don't already know :D